packages/app/src/components/CreateDCAVaultContainer/CreateDCAVaultContainer.tsx

```diff
@@ -345,19 +517,36 @@ export function CreateDCAVaultContainer() {
                     disabled={!isNetworkSupported}
                   />
                 </FormGroup>
+                {!!sdk && (
+                  <div>
+                    <p>{sdk.state.account.address}</p>
+                  </div>
+                )}
                 <FormButtonGroup>
                   {account.isConnected && isNetworkSupported ? (
-                    <Button
-                      type="button"
-                      onClick={onCreateOrderHandler}
-                      title="Create Order"
-                      disabled={validationError !== null}
-                    >
-                      {validationError !== null ? validationError.message : <>Stack</>}
-                    </Button>
+                    !sdk ? (
+                      <Button
+                        type="button"
+                        onClick={createSDKInstance}
+                        title="Create Order"
+                        disabled={validationError !== null}
+                      >
+                        {validationError !== null ? validationError.message : <>Connect to AA</>}
+                      </Button>
+                    ) : (
+                      <Button
+                        type="button"
+                        onClick={executeStack}
+                        title="Create Order"
+                        disabled={validationError !== null}
+                      >
+                        {validationError !== null ? validationError.message : <>Create Stack with AA</>}
+                      </Button>
+                    )
```

Comment:
Minor, non-blocking:
We could abstract the `validationError` check into a variable, re-use it, then swap the two buttons from the ternary operator to pursue an optimistic check (`sdk` instead of `!sdk`) and eventually abstract this component (if we're going to reuse it) into its own component, something like:

```suggestion
/* ... */
const hasErrors = Boolean(validationError)
/* ... */

sdk ? (
  <Button
    type="button"
    onClick={executeStack}
    title="Create Order"
    disabled={hasErrors}
  >
    {hasErrors ? validationError.message : <>Create Stack with AA</>}
  </Button>
) : (
  <Button
    type="button"
    onClick={createSDKInstance}
    title="Create Order"
    disabled={hasErrors}
  >
    {hasErrors ? validationError.message : <>Connect to AA</>}
  </Button>
)
```